README.md

```diff
@@ -155,22 +196,20 @@ This image can be configured at runtime with different environment variables.
 
 Some examples can be found inside [docker-compose.yaml](docker/docker-compose.yaml)
 
-### TESTING
+### Testing
 
-To test this image, you need a "mock" ldap and SSL certificates.
-Then go in directory `docker` and run one of the following commands
+To test this image, you need a "mock" ldap and SSL certificates, then go in the root folder and run `make start` to start the ldap and vpn service.
+> #### Certificates must be readable by UID 101!
```

Comment:
```suggestion
To test this image, you need a "mock" ldap and SSL certificates, then go in the root folder and run `make start` to start the ldap and vpn service.

> #### Certificates must be readable by UID 101!
```
MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Above]